aiven-python/__main__.py

```diff
@@ -0,0 +1,25 @@
+import pulumi
+import pulumi_aiven as aiven
+
+kafka1 = aiven.Kafka("kafka1",
```

Comment:
Similarly to https://github.com/pulumi/templates/pull/480#discussion_r1067575686:

```suggestion
# Create a Kafka service.
kafka = aiven.Kafka("kafka",
```